Makefile

```diff
@@ -84,13 +84,10 @@ race_condition_run: race_condition_binary
 search_index:
 	godoc -index -write_index -index_files='search_index'
 
-test: dependencies tools web
+test: dependencies tools
 	$(GO) test $(GO_TEST_FLAGS) ./...
 
 tools: dependencies
 	$(MAKE) -C tools
 
-web: dependencies
```

Comment:
Why not leave this in as a convenience? If you don't depend on it it won't be called by default.